test-integration/helpers/describe.coffee

```diff
@@ -49,12 +51,33 @@ describe = (name, cb) ->
       # Wait 20sec for the browser to start up
       @timeout(20 * 1000, true)
 
+      @downloadDirectory = "#{__dirname}/".replace('helpers/', 'downloaded-files')
+
+      chromeOptions = new chrome.Options()
+      prefs = {
+        download:
+          "default_directory": @downloadDirectory
+      }
+      chromeOptions.setUserPreferences(prefs)
+
+      firefoxProfile = new firefox.Profile()
+      firefoxProfile.setPreference("browser.helperApps.neverAsk.saveToDisk", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+      firefoxProfile.setPreference("browser.helperApps.neverAsk.openFile", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+      firefoxProfile.setPreference("browser.helperApps.alwaysAsk.force", false)
+      firefoxProfile.setPreference("browser.download.useDownloadDir", true)
+      firefoxProfile.setPreference("browser.download.manager.showWhenStarting",false)
+      firefoxProfile.setPreference("browser.download.folderList", 2)
+      firefoxProfile.setPreference("browser.download.dir", @downloadDirectory)
```

Comment:
ooh, nice! thanks for setting a temp download dir (& working in FF)! Should it get cleared before the tests start?